pandapipes/std_types/std_type_class.py

```diff
@@ -67,88 +172,78 @@ def get_pressure(self, vdot_m3_per_s):
         assumed.
 
         :param vdot_m3_per_s: Volume flow rate of a fluid in [m^3/s]. Abs() will be applied.
```

Comment:
As far as I understand it, abs(vdot_m3_per_s) is not applied. Instead, bypassing is assumed for reverse flows

Reply:
Removed it as your point was already explained in the sentence before.